src/brep_faceter.cc

```diff
@@ -289,8 +286,8 @@ void brep_faceter(std::string brep_file, std::string json_file,
   read_metadata(json_file, materials_map);
 
   MBTool mbtool;
-  // TODO: Use faceting tolerance value in set_tags and review use of GEOMETRY_RESABS
-  mbtool.set_tags();
+  // TODO: review use of GEOMETRY_RESABS
+  mbtool.set_faceting_tol_tag(facet_tol.tolerance);
```

Comment:
This includes a change in behaviour, which should ideally be in a separate commit from refactoring.
(Personally, I wouldn't have changed the behaviour, but that's because I don't know how the output is being used.)